xargs/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include <cstdio>
+#include <string>
+#include <vector>
+#include <sys/wait.h>
+#include <unistd.h>
+
+int main(int argc, char* argv[]) {
+
+    const char * path = "/usr//bin/ls";
+    char *args[] = {argv[0], argv[1], argv[2], nullptr};
+    execv(path, args);
```

Comment:
`xargs` command is a general purpose command. But you always execute `/usr/bin/ls` with arguments within `argv`. 
Read the requirements of the homework carefully, it says you should execute command in `argv[1] `with arguments in `stdin`.